cmake/dependencies/PICSAR.cmake

```diff
@@ -41,10 +61,10 @@ endfunction()
 
 if(WarpX_QED)
     option(WarpX_picsar_internal   "Download & build PICSAR" ON)
-    set(WarpX_picsar_repo "https://github.com/ECP-WarpX/picsar.git"
+    set(WarpX_picsar_repo "https://github.com/lucafedeli88/picsar.git"
         CACHE STRING
         "Repository URI to pull and build PICSAR from if(WarpX_picsar_internal)")
-    set(WarpX_picsar_branch "development"
+    set(WarpX_picsar_branch "improve_makefile"
```

Comment:
Memo to ourselves: let's not forget to wait for the upstream PR to be merged and then revert this change here